.gitlab-ci.yml

```diff
@@ -49,6 +49,23 @@ deploy review:
   tags:
     - deploy
 
+# Deploy release to connect.trezor.io
+deploy to connect.trezor.io:
+  stage: deploy
+  only:
+    refs:
+      - v8
+  when: manual
+  variables:
+    GIT_STRATEGY: none
+  before_script:
+    - make clean
+    - make build
```

Comment:
No reason to have this in `before_script`, we can have it in `script` directly, nope? Yes, bikesheding 101.